### ISSUE TITLE
Change naming convention for config variables

### ISSUE THREAD

**Issue by cringoleg** [MarkusBordihn/BOs-Easy-Mob-Farm#42]
This is a description of my experience configuring the mod, which I tried after `6.6.2` update. 
The problem occurred when I updated the mod on my server - cows stopped dropping `beef`. 

In wiki for [drop item configuration](https://github.com/MarkusBordihn/BOs-Easy-Mob-Farm/wiki/Configuration#additional-drop-settings-) we see two settings for drops:
> `cowDropRawBeef`: A boolean value that determines whether cows should drop raw beef.
> `witherDropNetherStar` A boolean value that determines whether the wither should drop a nether star.

Which can be interpreted as:
> Will cow drop beef? - true/false
> Will wither drop nether star? - true/false

This is completely fine.

However, in config file comments *and* in [source code](https://github.com/MarkusBordihn/BOs-Easy-Mob-Farm/blob/6906ecbd3d0b3487b767eee69af06faefbaf6ac9/src/main/java/de/markusbordihn/easymobfarm/config/CommonConfig.java#L362) they mean completely opposite things:
The cow loot drop behavior:
``` java
    public final ForgeConfigSpec.BooleanValue disableCowDropRawBeef;
```
The wither loot drop behavior:
``` java
    public final ForgeConfigSpec.BooleanValue forceWitherDropNetherStar;
```

Config file comments:
- For cow
```
	["Additional Drop Settings"."Cow Drop Settings"]
		#Disable cow raw beef drops.
		cowDropRawBeef = false
```
- For wither
```
	["Additional Drop Settings"."Wither Drop Settings"]
		#Enable/Disable forced nether star drops from wither.
		witherDropNetherStar = false

```
To summarize: 
It was not obvious from config file comments and wiki combined, that I have to `disable` cow drops and `enable` the wither drops considering the variable names are built identical.

**Comment by cringoleg** [comment]
Nevermind, it was updated yesterday. :D
